ipyleaflet/leaflet.py

```diff
@@ -536,13 +536,34 @@ class GeoJSON(FeatureGroup):
     style = Dict().tag(sync=True)
     hover_style = Dict().tag(sync=True)
     point_style = Dict().tag(sync=True)
+    style_callback = Any()
```

Comment:
We could put a custom validator on this trait (see [traitlets validation](https://traitlets.readthedocs.io/en/stable/using_traitlets.html#validation)). That would validate that it's a callable (functor/function/lambda) using [the built-in callable function](https://docs.python.org/3/library/functions.html#callable).

I'm fine if we do it later though.

Reply:
I have added validation for style_callback. Thanks